### ISSUE TITLE
Add Roccat Tyon Mouse

### ISSUE THREAD

**Issue by fugenkleber** [denilsonsa/udev-joystick-blacklist#22]
Please add the Roccat Tyon Mouse to the blacklist. It's recognized as a joystick on Xubuntu 16.10.

`lsusb | grep ROCCAT`
`Bus 001 Device 002: ID 1e7d:2e4a ROCCAT`

**Comment by denilsonsa** [comment]
Thanks!

**Comment by SimonDedman** [comment]
Hi guys, regarding this: I have the same mouse & issue; if I use this script won't it mean the mouse isn't recognised therefore I can't use it? That's how I interpret the description in the readme, please let me know if not. Thanks!

**Comment by denilsonsa** [comment]
@SimonDedman, hi!

**TL;DR:** Your mouse should still work fine. Only the joystick part is disabled, the rest should continue working.

Some hardware/physical devices are recognized as multiple software/logical devices. For instance, a keyboard that has a built-in touchpad is one hardware device, but it generates two software devices: one for the keyboard and one for the mouse/touchpad.

The scripts in this repository work by disabling or removing the "joystick" (AKA gamepad) software device from the blacklisted hardware. Thus, a Microsoft keyboard will only be recognized as a keyboard, and not as a keyboard+"fake-joystick-that-does-not-work".

**Comment by SimonDedman** [comment]
Nice one, cheers amigo!
